espanol/torre/huida/huida.md

```diff
@@ -1 +1,3 @@
 No se puede medir la fuerza del enemigo y su estimacion supone una fuerza muy alta.
+
+Por lo que el consejero decidió de nuevo [subir a la torre](torre/torre.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.